content/api/commands/selectfile.md

```diff
@@ -0,0 +1,276 @@
+---
+title: selectFile
+---
+
+Selects a file or files in an HTML5 input element or simulates dragging a file
+or files into the browser.
+
+## Syntax
+
+```javascript
+.selectFile(file)
+.selectFile(file, options)
+.selectFile([file1, file2, ...])
+.selectFile([file1, file2, ...], options)
+```
+
+### Usage
+
+**<Icon name="check-circle" color="green"></Icon> Correct Usage**
+
+```javascript
+cy.get('input[type=file]').selectFile('file.json')
+cy.get('input[type=file]').selectFile(['file.json', 'file2.json'])
+cy.get('input[type=file]').selectFile({
+  contents: Buffer.from('foo'),
+  fileName: 'file.json',
+  lastModified: Date.now(),
+})
+```
```

Comment:
Can we show one or more drag+drop examples here, eg.

```js
cy.get('.file-target').selectFile('file.json', { action: 'drag-drop' })
cy.document().selectFile('file.json', { action: 'drag-drop' })
```

Reply:
Added.